server/containers/rpmrepo/processincoming.sh

```diff
@@ -58,6 +59,7 @@ while read -r -u 10 pkgname; do
         if [ -n "${old_rpms}" ]; then
             rm "${old_rpms}"
         fi
+        remove_old_dev_pkg.py --keep=3 ${local_dist}
```

Comment:
You also need to update the database, simply removing the RPMs isn't enough.

Reply:
Isn't the database updated for me below in `createrepo --update "${local_dist}"`?

Reply:
Yeah, I think so. `man repomanage` says `repomanage - list the newest or oldest RPM packages in a directory` and:

> `repomanage` is a program to manage a directory of RPM packages. It displays a list of the newest or oldest packages in a directory for easy piping to xargs or similar programs.

Reply:
Ignore me, the update is done below outside of the condition.